lib/promscrape/discovery/ovhcloud/vps.go

```diff
@@ -0,0 +1,173 @@
+package ovhcloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/netip"
+	"net/url"
+	"path"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/promutils"
+)
+
+// vpsModel struct from API.
+type vpsModel struct {
```

Comment:
Would be great to have link to the OVH docs with response fields

Reply:
same as above